driven/storage/database.go

```diff
@@ -191,6 +199,18 @@ func (m *database) applyCategoriesChecks(categories *collectionWrapper) error {
 	return nil
 }
 
+func (m *database) applyMetaDataChecks(metaData *collectionWrapper) error {
+	log.Println("apply meta_data checks.....")
+	//Add org_id + app_id index
+	err := metaData.AddIndex(bson.D{primitive.E{Key: "org_id", Value: 1},
```

Comment:
You do not ahve these fields in the model which you store in the database but you put indexes on them? YOu have to put unique index on the _key_ field.